modules/aws-http4s/src/smithy4s/aws/internals/AwsRestJsonCodecs.scala

```diff
@@ -52,8 +53,13 @@ private[aws] object AwsRestJsonCodecs {
           .andThen[HttpMediaReader](HttpMediaTyped.mediaTypeK(mediaType))
       }
 
+    def nullToEmptyObject(blob: Blob): Blob =
+      if (blob.sameBytesAs(Blob("null"))) Blob("{}") else blob
```

Comment:
We could have a `Blob.NullString` in place of `Blob("null")`

Reply:
Well `Blob` ain't supposed to be biased towards Json but we can certainly have a `JsonNullBlob` constant in the Json module :+1:

Reply:
[1506755](https://github.com/disneystreaming/smithy4s/pull/1094/commits/15067558e2834867f1f18c50b0594ff1e9f83c5b)